tests/step-103.sh

```diff
@@ -83,7 +83,7 @@ fi
 # Clone a new one
 echo "Cloning"
 cd /tmp || exit 1
-git clone /tmp/test103 test103-clone && cd test103-clone || exit 1
+GIT_TRACE=1 git clone /tmp/test103 test103-clone && cd test103-clone || exit 1
```

Comment:
What does `GIT_TRACE=1` do?

Reply:
Ups left over. From Debugging, it just outputs Traces... quite handy..